Add support for Sis Imports (model, impl and multipart methods)

bgarciaentornos opened the pull request.

buckett commented on src/main/java/edu/ksu/canvas/requestOptions/CreateSisImportOptions.java:
```diff
@@ -0,0 +1,38 @@
+    private final String filePath;
+    private final ImportType importType;
+    
+    public CreateSisImportOptions(String accountId, String filePath, ImportType importType) {
```
It would be helpful if there was an option to supply an `InputStream` or something similar. That way the calling code doesn't have to write a temp file out to disk (although it may wish to anyway).

bgarciaentornos replied:
Do you mean an additional parameter (which might or not be used), or replacing the filePath param altogether? The code where this will be used is reading from a file, which then comes handy for the FileBody Multipart bit on the SimpleRestClient. The alternative would be using the raw binary body way. Maybe I should create different methods for both ways?

buckett replied:
Yeah, I think there's value in both being able to supply a file, but also being able to hand it an `InputStream`

bgarciaentornos commented:
I have added an InputStream parameter as requested, only used if not null. Also added another param ('diffing_data_set_identifier') that I need for the DP-cloud project.
